apps/nextjs-website/src/components/atoms/BodyWrapper/BodyWrapper.tsx

```diff
@@ -2,18 +2,18 @@
 import React from 'react';
 import { useTheme } from '@mui/material';
 
-const MainWrapper = ({ children }: { children: React.ReactNode }) => {
+const BodyWrapper = ({ children }: { children: React.ReactNode }) => {
   const { palette } = useTheme();
 
   return (
-    <main
+    <body
       style={{
-        backgroundColor: palette.background.paper,
+        backgroundColor: palette.common.white,
```

Comment:
Just a curiosity: why the `backgroundColor` has been change from `palette.background.paper` to `palette.common.white`?

I mean, I'm not thinking the previous value was better, but semantically I think it was more appropriate. So, I'd like to know the ratio behind this change (of course both `palette.background.paper` and `palette.common.white` are the same color).

Reply:
Just an oversight